### PR TITLE
build!: require node v20

### PR DESCRIPTION
**Pull request by viceice**
## Changes
Require node v20 (with latest security fixes)

## Context
- #27992

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Review by secustor — APPROVED** [review]

File: docs/development/local-development.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ For example, if you think anything is unclear, or you think something needs to b
 You need the following dependencies for local development:
 
 - Git `>=2.45.1`
-- Node.js `^18.12.0 || >=20.0.0`
+- Node.js `^20.15.1 || >=22.5.1`
```
Comment (rarkins) [inline docs/development/local-development.md:14]:
```suggestion
- Node.js `^20.15.1`
```

File: package.json [hunk 2]
```diff
@@ -134,13 +134,9 @@
   },
   "homepage": "https://renovatebot.com",
   "engines": {
-    "node": "^18.12.0 || >=20.0.0",
+    "node": "^20.15.1 || >=22.5.1",
```
Comment (rarkins) [inline package.json:137]:
```suggestion
    "node": "^20.15.1",
```

**Review by rarkins — CHANGES_REQUESTED** [review]

**Review by rarkins — APPROVED** [review]

**Review by SamyDgz — APPROVED** [review]

**Review by Goop-arsh — COMMENTED** [review]
Run

**Comment by Goop-arsh** [comment]
R

**Pull request merged**